zephyr/module.yml

```diff
@@ -0,0 +1,4 @@
+build:
+  settings:
+    soc_root: .
+    board_root: . 
```

Comment:
board root should only be used in tests, not generally available. These boards should not appear in `west boards` or be usable in normal `west build -b...`

Reply:
thx, missed removal